frontend/src/app/services/event-list.ts

```diff
@@ -1,49 +1,71 @@
 import {DayState} from './simulation';
 import {EventTrigger} from './events';
 import {dateDiff} from './utils';
-import {formatNumber} from '../utils/format';
+import _ from 'lodash';
```

Comment:
Please, import just the specific function. It should work better for tree shaking of the code (just used functions get into the resulting code).

```suggestion
import {random} from 'lodash';
```